locale/en/foreman_plugin_template.po

```diff
@@ -0,0 +1,19 @@
+# foreman_plugin_template
```

Comment:
Ah, sorry, I refetched the en directory from the plugin template.  I'll remove it.